test/api/controllers/writeController.test.js

```diff
@@ -30,6 +30,7 @@ describe('Test: write controller', function () {
     should(kuzzle.funnel.write.create(requestObject)).be.rejected();
     should(kuzzle.funnel.write.createOrUpdate(requestObject)).be.rejected();
     should(kuzzle.funnel.write.update(requestObject)).be.rejected();
+    should(kuzzle.funnel.write.replace(requestObject)).be.rejected();
```

Comment:
Hm, well, this is awkward: this test cannot work as it is.
ShouldJS only test the result of promises when returning them, like this:
```js
return should(<some method returning a promise>).be.rejected();
```

So this test must be split into 4 distinct tests to work.